src/main.scss

```diff
@@ -1,27 +1,78 @@
-@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap');
```

Comment:
oops i did not specify that we use prettier for everything Except css

Reply:
I'll revert it, although I think prettier does a pretty good job here. The only contentful change is
`@@ -263,6 +317,11 @@ button {`
```      &[href],
      &[tabindex] {
        cursor: pointer;
      }